accounts/abi/event_test.go

```diff
@@ -17,13 +17,45 @@
 package abi
 
 import (
+	"encoding/hex"
+	"encoding/json"
+	"math/big"
 	"strings"
 	"testing"
 
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/stretchr/testify/assert"
```

Comment:
You're pulling in two libraries that produce a big diff (and code) size, and you're the only one using it. I suggest that you create a different PR to start using those two libraries, so that the content of this PR is only about `tupleUnpack`. If the community is interested in using testify, you will get upvotes and we'll then include it. This PR does too many different things and we can't include it whole.

Reply:
OK, will do that.

Reply:
Testify is a really convenient library when doing tests. I will extract it from this pull request and do another one then.